header.php

```diff
@@ -75,7 +75,7 @@
 	do_action( 'neve_before_header_wrapper_hook' );
 	?>
 
-	<header class="<?php echo esc_attr( $header_classes ); ?>" role="banner" next-page-hide>
+	<header class="<?php echo esc_attr( $header_classes ); ?>" next-page-hide>
```

Comment:
Let's use `neve_is_amp` function to display the `next-page-hide` just in an AMP context.